### PR TITLE
AutoWorld: add preliminary .apworld specification

### PR DESCRIPTION
**Pull request by black-sliver**

### CODE REVIEW

File: docs/apworld specification.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# apworld Specification
+
+Archipelago depends on worlds to provide game-specific details like items, locations and output generation.
+Those are located in the `worlds/` folder (source) or  `<insall dir>/lib/worlds/` (when installed).
+See [world api.md](world api.md) for details.
+
+apworld provides a way to package and ship a world that is not part of the main distribution by placing a `*.apworld`
+file in the worlds folder.
```
Comment (black-sliver) [inline docs/apworld specification.md:8]:
```suggestion
file into the worlds folder.
```

**Comment by Berserker66** [comment]
Maybe this should target main instead; then both PRs can get squashed and authorship for each of the two squash commits is clean?

**Comment by Berserker66** [comment]
currently "files changed" shows all the changes from the other branch as well.

**Comment by black-sliver** [comment]
that's ... unfortunate. gimme a minute

**Comment by Berserker66** [comment]
if we merge the other first, then squash this one, it should work?

**Comment by black-sliver** [comment]
Fixed

**Review by Berserker66 — APPROVED** [review]

**Pull request merged**